README.md

```diff
@@ -44,24 +44,38 @@ Fly is extremely extensible, so _anything_ can be a task. Our core system will a
 <h2>Table of Contents</h2>
 <details>
 <summary>Table of Contents</summary>
-- [Features](#features)
```

Comment:
I am pretty sure it worked earlier, seems like some updates to GitHub's markdown parsing. Can you just add new line here. It is not convenient to use lots of tags inside markdown.